Makefile

```diff
@@ -7,10 +7,12 @@ all: openwrt
 openwrt: build_openwrt
 	echo "OpenWrt Done!"
 
-clean: clean_openwrt clean_feeds clean_binaries
+clean: clean_openwrt clean_feeds clean_binaries clean_keys
 
 # Building OpenWRT
 $(DIR__CI)/patched:
+	/vault read -field=key secret/creator/packagesigning > $(DIR__OPENWRT)/key.pem
+	/vault read -field=cert secret/creator/packagesigning > $(DIR__OPENWRT)/cert.pem
```

Comment:
Please move the vault instructions after ./scripts/feeds install -a step. The build is failing with the following error due to current sequence

    /vault read -field=cert secret/creator/packagesigning > <http://10.100.124.153:8080/job/Build_All_CA_PR/ws/openwrt/cert.pem>
    git submodule init openwrt;git submodule update --remote; \
	    cd <http://10.100.124.153:8080/job/Build_All_CA_PR/ws/openwrt;> \
	    ./scripts/feeds update -a; \
	    ./scripts/feeds install -a;
    Submodule 'openwrt' (https://github.com/CreatorDev/openwrt.git) registered for path 'openwrt'
    fatal: destination path 'openwrt' already exists and is not an empty directory.
    Clone of 'https://github.com/CreatorDev/openwrt.git' into submodule path 'openwrt' failed
    /bin/sh: 3: ./scripts/feeds: not found
    /bin/sh: 4: ./scripts/feeds: not found